### PR TITLE
[ROS][Test Infra] Support testing ROS2 service call device_info

### PR DESCRIPTION
**Pull request by kadiredd**
Implementation of JIRA ticket req: [LRS-1122]

### CODE REVIEW

File: realsense2_camera/test/utils/pytest_rs_utils.py [hunk 1]
```diff
@@ -833,7 +837,13 @@ def set_bool_param(self, param_name, param_value):
         new_param_value = ParameterValue(type=ParameterType.PARAMETER_BOOL, bool_value=param_value)
         req.parameters = [Parameter(name=param_name, value=new_param_value)]
         return self.send_param(req)
-
+    
+    def get_bool_param(self, param_name):
+        req = GetParameters.Request()
+        req.names = [param_name]
+        value = self.get_param(req)
```
Comment (Arun-Prasad-V) [inline realsense2_camera/test/utils/pytest_rs_utils.py:844]: please add a check to verify whether the response is not None.

Comment (kadiredd) [reply realsense2_camera/test/utils/pytest_rs_utils.py:844]: fixed

File: realsense2_camera/test/live_camera/test_camera_service_call.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+'''
+The test is to check service call device_info with type DeviceInfo
+'''
+@pytest.mark.parametrize("launch_descr_with_parameters", [    
+    pytest.param(test_params_test_srv_d455, marks=pytest.mark.d455),
+    pytest.param(test_params_test_srv_d435i, marks=pytest.mark.d435i),
+    pytest.param(test_params_test_srv_d415, marks=pytest.mark.d415),    
+    ],indirect=True)
+@pytest.mark.launch(fixture=launch_descr_with_parameters)
+class TestCamera_ServiceCall(pytest_rs_utils.RsTestBaseClass):
```
Comment (Arun-Prasad-V) [inline realsense2_camera/test/live_camera/test_camera_service_call.py]: this test only uses DeviceInfo service call. So, shall we rename this function and file names such that it is specific to DeviceInfo service.

Comment (kadiredd) [reply realsense2_camera/test/live_camera/test_camera_service_call.py]: Yes, thought of it already - There are 8 service calls available. Plan is to have rest seven of them be added here in this file, so no need of changing the file name as of now. And, about changing the function name, yes/maybe -- to be addressed when implementing rest of the service calls. This is kind of placeholder test for validating service calls.

Comment (kadiredd) [reply realsense2_camera/test/live_camera/test_camera_service_call.py]: got it, each test will carry distinct function name and distinct class name as well, right? Modified both class and function names.

**Review by Arun-Prasad-V — APPROVED** [review]
LGTM.

**Review by SamerKhshiboun — APPROVED** [review]
LGTM, but don't we prefer to rename the file name to device info service ? or we want to add all services tests in the future in the same file ?

**Comment by kadiredd** [comment]
> LGTM, but don't we prefer to rename the file name to device info service ? or we want to add all services tests in the future in the same file ?

We want to add all other services in the same file, Yes.

**Pull request merged**